initial-report/index.html

```diff
@@ -43,6 +43,139 @@
     </p>
   </section>
   <section id="sotd"></section>
+  <section class="informative" id="foreword">
+    <h2>Foreword</h2>
+    <p>This report is still in the process of being written. Please see the <a
+        href="https://github.com/swicg/activitypub-trust-and-safety/issues/32">Initial Report issue</a> on GitHub for
+      current status.
+    </p>
+  </section>
+  <section>
+    <h2>Introduction</h2>
+    <p>ActivityPub and ActivityStreams are inherently social protocols, as such, they must adequately address trust
+      and
+      safety
+      for the success of the protocol and the platforms that built on top. At <time title="early 2025">present</time>,
+      ActivityPub as a protocol does not do enough to ensure the safety of people
+      using the platforms and trust between users and platform operators. We have previously seen issues with
```

Comment:
Remove:
> At present, ActivityPub as a protocol does not do enough to ensure the safety of people using the platforms and trust between users and platform operators.

Replace with something explaining that reading the ActivityPub and ActivityStreams specifications alone won't give you enough information for building activitypub software with good trust and safety.